src/Plugin/VoteUpDownWidgetInterface.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace Drupal\vud\Plugin;
+
+use Drupal\Component\Plugin\PluginInspectionInterface;
+
+/**
+ * Defines an interface for Vote Up/Down Widget plugins.
+ */
+interface VoteUpDownWidgetInterface extends PluginInspectionInterface {
+
+  /**
+   * Method to return the label of the plugin whenever required
+   *
+   * @return mixed
+   */
+  public function getLabel();
+
+  /**
+   * Method to alter template variables on render
```

Comment:
A space is needed after the summary line on the docblock.
This change applies to several places on this commit, please fix them.